examples/ecommerce-netlify-functions/test/checkout.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { describe, it, before, after } = require('mocha');
+const assert = require('assert');
+const { handler: addToCart } = require('../functions/addToCart');
+const { handler: checkout } = require('../functions/checkout');
+const mongoose = require('mongoose');
+
+
+
+describe('Checkout', function() {
+  before(async() => {
+    await mongoose.connect('mongodb://localhost:27017/netlify');
+  });
+
+  after(async() => {
+    await mongoose.disconnect();
+  });
+  it('Should do a successful checkout run', async function() {
+    const params = {
+      body: {
+        cartId: null,
+        product: [
+          { productId: '629e5f3686d82fc73b95b396', quantity: 2 },
+          { productId: '629e5f3686d82fc73b95b398', quantity: 1 }
+        ]
+      }
+    };
+    const result = await addToCart(params);
+    assert(result.body);
+    assert(result.body.items.length);
+    params.body.cartId = result.body._id;
+    const finish = await checkout(params);
```

Comment:
Use [sinon to stub out Stripe](https://masteringjs.io/tutorials/sinon/stub). Tests ideally shouldn't rely on external APIs or services, outside of MongoDB.